evm/generators.go

```diff
@@ -1374,6 +1374,17 @@ func {{.DeployHandler.HandlerName}}() *cobra.Command {
 				} else {
 					copy(salt[:], safeSaltRaw)
 				}
+
+				if safeNonceRaw == "" {
+					fmt.Println("--safe-nonce not specified, using default (0)")
+					safeNonce = big.NewInt(0)
+				} else {
+					safeNonce = new(big.Int)
+					_, ok := safeNonce.SetString(safeNonceRaw, 10)
```

Comment:
Better to use `0` instead of `10`. That will also accept hex input, etc.

https://pkg.go.dev/math/big#Int.SetString